scripts/src/primazactl/cmd/create/namespace/common.py

```diff
@@ -72,6 +73,16 @@ def add_args_namespace(parser: argparse.ArgumentParser, type):
                 on which Primaza's Control Plane is running",
         default=None)
 
+    parser.add_argument(
+        "-j", "--service-account-namespace",
+        dest="service_account_namespace",
+        required=False,
+        help=f"namespace used for hosting the service account shared with\
+             Primaza's Control Plane(existing namespace).\
+             Default: {WORKER_NAMESPACE}.",
+        type=kubernetes_name,
+        default=None)
+
```

Comment:
This value should be required only in the `join cluster` command. not in create `create *-namespace`.

Reply:
I can see, you're right